File: C/factorial.c
#include <stdio.h>

int main() 
{
    int number;
    long factorial = 1;

    printf("Input the number: ");
    scanf("%d", &number);

    if (number < 0) 
    {
        printf("Factorial is not defined for negative numbers.\n");
    }
    for (int i = 1; i <= number; ++i) 
    {
        factorial *= i;
    }
    printf("The Factorial of %d is: %d\n", number, factorial);
    return 0;
}

File: JavaScript/factorial.js
function factorialOfNumber() {
    console.log(" Factorial of a Number (5)");
    var number = 5;
    var factorial = 1;
    for (var i = 1; i <= number; i++) {
        factorial *= i; 
    }
    console.log("Factorial of " + number + " is: " + factorial);
